Source/core/JSON.h

```diff
@@ -1938,17 +1949,66 @@ namespace Core {
                 REPLACE
             };
 
+            char GetEscapeSequenceValue(char current) const
+            {
+                 char value = 0;
+                 switch(current) {
+                 case 'b':
+                     value = 0x08;
+                     break;
+                 case 'f':
+                     value = 0x0c;
+                     break;
+                 case 'n':
+                     value = 0x0a;
+                     break;
+                 case 'r':
+                     value = 0x0d;
+                     break;
+                 case 't':
+                     value = 0x09;
+                     break;
+                 default:
+                     break;
+                 }
+                 return value;
+            }
+
             EscapeSequenceAction GetEscapeSequenceAction(char current) const
             {
                 EscapeSequenceAction action = EscapeSequenceAction::COLLAPSE;
                 if (current == 'u' || current == 'n' || current == 't' || current == 'r' || current == 'f' || current == 'b') {
```

Comment:
Can we all move this to 1 array, cause now the data is a bit distributed..
So create a LUT like: { { 'b', 0x08 }, { 'f', 0x0c }, ...} and than use it in the GetEscapeSequence/IsEscapeSequence but also here, that you check if the character IsEscapeCharacter, since I think there is now a mismatch, all conversions have b,f,n,r,t but no u and here is a check for the u

Reply:
Ok, will update

Reply:
Added Lookup Map to handle this